pkg/aws/policies.go

```diff
@@ -1602,11 +1608,15 @@ func (c *awsClient) ValidateOperatorRolesManagedPolicies(cluster *cmv1.Cluster,
 }
 
 func (c *awsClient) ValidateAccountRolesManagedPolicies(prefix string, policies map[string]*cmv1.AWSSTSPolicy) error {
-	for key, accountRole := range AccountRoles {
+	for roleType, accountRole := range AccountRoles {
```

Comment:
I feel like we should separate policies.go into two files for managed/unmanaged

Reply:
We can do that.
Anyhow, we can do it in a following PR, to keep this one concise.